### PR TITLE
Feat/create movie detals

### PR DESCRIPTION
**Pull request by ArmanKhoyetsyan**
Аdded all components : title , information , movie-rating, movie-overview , movie-writer,movie-actor

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
     text-align: center;
     display: flex;
     justify-content: center;
+    background-color:#E5E5E5;
```
Comment (M7Arman) [inline src/App.scss]: variable.scss

File: src/componets/movie-details/movie-actor/create-movie-actor/create-movie-actor.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import "./create-movie-actor.scss"
+import imgSrc from '../../../../assets/img/Ellipse 1.svg'
```
Comment (M7Arman) [inline src/componets/movie-details/movie-actor/create-movie-actor/create-movie-actor.js]: Don't have space in filename

File: src/componets/movie-details/movie-actor/movie-actor.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import CreateMovieActor from './create-movie-actor/create-movie-actor'
+import './movie-actor.scss'
+
+function MovieActor(){
+    return(
+        <div className="movieActor">
+            <CreateMovieActor/>
+            <CreateMovieActor/>
+            <CreateMovieActor/>
+            <CreateMovieActor/>
```
Comment (M7Arman) [inline src/componets/movie-details/movie-actor/movie-actor.js]: Have an array of actors data and show all of them with passing data via props

File: src/componets/movie-details/movie-information/movir-information.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+ import "./movir-information.scss"
```
Comment (M7Arman) [inline src/componets/movie-details/movie-information/movir-information.js]: Fix typo in filename

**Comment by ArmanKhoyetsyan** [comment]
https://github.com/dwintechnology/Gryffindor/pull/12

fixed all issues

**Pull request merged**